src/Microsoft.AspNet.Diagnostics/Views/CompilationErrorPage.cshtml

```diff
@@ -45,33 +45,27 @@
                             <h3>@frame.ErrorDetails</h3>
                         }
 
-                        @if (frame.Line != 0 && frame.ContextCode !=null && frame.ContextCode.Any())
+                        @if (frame.Line != 0 && frame.ContextCode != null && frame.ContextCode.Any())
                         {
                             <div class="source">
-                                @if (frame.PreContextCode != null)
-                                {
-                                    <ol start="@frame.PreContextLine" class="collapsible">
-                                        @foreach (var line in frame.PreContextCode)
-                                        {
-                                            <li><span>@line</span></li>
-                                        }
-                                    </ol>
-                                }
+                                <ol start="@frame.PreContextLine" class="collapsible">
+									@foreach (var line in frame.PreContextCode)
+									{
+										<li><span>@line</span></li>
+									}
+								</ol>
```

Comment:
weird...i used Notepad++(as opening in VS was causing other unrelated changes) for this in which it was ok...will fix..